sdk/examples/xo_go/src/sawtooth_xo/handler/handler.go

```diff
@@ -0,0 +1,321 @@
+			return true
+		}
+	}
+	return false
+}
+
+func unpackPayload(payloadData []byte) (*XoPayload, error) {
+	if payloadData == nil {
+		return nil, &processor.InvalidTransactionError{"Must contain payload"}
+	}
+
+	parts := strings.Split(string(payloadData), ",")
+	if len(parts) != 3 {
+		return nil, &processor.InvalidTransactionError{"Payload is malformed"}
+	}
+
+	payload := XoPayload{}
+	payload.Name = parts[0]
+	payload.Action = parts[1]
+
+	if len(payload.Name) < 1 {
+		return nil, &processor.InvalidTransactionError{"Name is required"}
+	}
+
+	if len(payload.Action) < 1 {
+		return nil, &processor.InvalidTransactionError{"Action is required"}
+	}
+
+	if payload.Action == "take" {
+		space, err := strconv.Atoi(parts[2])
+		payload.Space = space
+		if err != nil {
+			return nil, &processor.InvalidTransactionError{
+				fmt.Sprintf("Invalid Space : '%v'", parts[2])}
+		}
+	}
+
+	return &payload, nil
+}
+
+func unpackHeader(headerData []byte) (*transaction_pb2.TransactionHeader, error) {
+	header := &transaction_pb2.TransactionHeader{}
+	err := proto.Unmarshal(headerData, header)
+	if err != nil {
+		return nil, &processor.InternalError{
+			fmt.Sprint("Failed to unmarshal TransactionHeader: %v", err)}
+	}
+	return header, nil
+}
+
+func unpackGame(gameData []byte) (*Game, error) {
+	parts := strings.Split(string(gameData), ",")
+	if len(parts) != 5 {
+		return nil, &processor.InternalError{
+			fmt.Sprintf("Failed to deserialize game data: '%v'", string(gameData))}
+	}
+
+	game := &Game{
+		Board:   parts[0],
+		State:   parts[1],
+		Player1: parts[2],
+		Player2: parts[3],
+		Name:    parts[4],
+	}
+
+	switch game.State {
+	case "P1-WIN", "P2-WIN", "TIE", "P1-NEXT", "P2-NEXT":
+	default:
+		return nil, &processor.InternalError{
+			fmt.Sprintf("Game has reached invalid state: '%v'", game.State)}
+	}
+
+	return game, nil
+}
+
+func packGame(game *Game) []byte {
+	var buffer bytes.Buffer
+	buffer.WriteString(game.Board)
+	buffer.WriteString(",")
+	buffer.WriteString(game.State)
+	buffer.WriteString(",")
+	buffer.WriteString(game.Player1)
+	buffer.WriteString(",")
+	buffer.WriteString(game.Player2)
+	buffer.WriteString(",")
+	buffer.WriteString(game.Name)
+	return buffer.Bytes()
+}
+
+func loadGame(name string, state *processor.State) (*Game, error) {
+	// Use the namespace prefix + the hash of the game name to create the
+	// storage address
+	address := namespace + hexdigest(name)
+
+	results, err := state.Get([]string{address})
+	if err != nil {
+		return nil, &processor.InternalError{fmt.Sprint("Error getting state:", err)}
+	}
+
+	if len(string(results[address])) > 0 {
+		game, err := unpackGame(results[address])
+		if err != nil {
+			return nil, err
+		}
+
+		// NOTE: Since the game data is stored in a Merkle tree, there is a
+		// small chance of collision. A more correct usage would be to store
+		// a dictionary of games so that multiple games could be store at
```

Comment:
(Grammar/typo): 
Change
 _a dictionary of games so that multiple games could be store at_
to 
 a dictionary of games so that multiple games could be **_stored_** at